examples/pixart_sigma/README.md

```diff
@@ -7,14 +7,13 @@ This repo contains Mindspore model definitions, pre-trained weights and inferenc
 - Main
     - [Training](#vanilla-finetune)
     - [Inference](#getting-start)
-    - [Use diffusers: coming soon]
-    - [Launch Demo: coming soon]
-- Guidance
-    - [Feature extraction: coming soon]
-    - [LoRA & DoRA: coming soon]
+    - [Launch Demo](#running-the-demo-with-gradio)
+    - [Use diffusers](#integration-in-diffusers)
 - Benchmark
     - [Training](#training)
     - [Inference](#inference)
+- Guidance
+    - [LoRA Finetuning](#)
```

Comment:
No introduction to lora finetune at below？

Reply:
I updated the new README (with an illustration of the Gradio demo) and removed the LoRA fine-tuning part. The LoRA fine-tuning code and tutorial will appear in another PR.